challenges/08-coding-interview-prep/project-euler.json

```diff
@@ -234,11 +234,21 @@
           "testString":
             "assert.strictEqual(smallestMult(5), 60, '<code>smallestMult(5)</code> should return 60.');"
         },
+        {
+          "text": "<code>smallestMult(7)</code> should return 420.",
+          "testString":
+            "assert.strictEqual(smallestMult(7), 420, '<code>smallestMult(5)</code> should return 420.');"
```

Comment:
Minor thing, but could we change this line to the following?:

`"assert.strictEqual(smallestMult(7), 420, '<code>smallestMult(7)</code> should return 420.');"`

Reply:
I'm sorry about that typo. Let me if there are any more changes to be made, I'll make the proposed changes and update the pull request.

Reply:
No need to apologize! It happens to the best of us. And it's such a minor detail I almost missed it myself.